launch/launch/launcher.py

```diff
@@ -39,6 +40,11 @@ def __init__(self, name_prefix='', sigint_timeout=10):
         self.sigint_timeout = sigint_timeout
         self.task_descriptors = []
         self.print_mutex = threading.Lock()
+        self.loop = None
+        self.loop_lock = threading.Lock()
+        self.interrupt_future = asyncio.Future()
+        self.launch_complete = threading.Event()
+        self.processes_launched = threading.Event()
```

Comment:
Maybe better ``processes_spawned` since *to spawn* is the commonly used verb for processes?